rmgpy/molecule/atomtype.py

```diff
@@ -249,8 +249,8 @@ def get_features(self):
                              benzene=[0], lone_pairs=[0])
 # Occupied surface site:
 ATOMTYPES['Xo']   = AtomType('Xo', generic=['X'], specific=[],
-                             single=[0, 1], all_double=[0, 1], r_double=[], o_double=[], s_double=[], triple=[0, 1],
-                             quadruple=[0, 1], benzene=[0], lone_pairs=[0])
+                             single=[0, 1, 2, 3, 4, 5, 6, 7, 8], all_double=[0, 1, 2, 3, 4, 5, 6, 7, 8], r_double=[], o_double=[], s_double=[], triple=[0, 1, 2, 3, 4, 5, 6, 7, 8],
```

Comment:
What does this mean? Do we allow up to 8 single bonds on surface site or something? Can you add comments to this change

Reply:
Yes, this makes it possible for the surface site to have up to 8 bonds...the intention is that you can represent local surface structure (what sites neighbor other sites) which affects adsorbate energies and reaction rates on surfaces.